db/migrate/20110216164102_add_facilities_journal_mask_column.rb

```diff
@@ -1,6 +1,6 @@
 class AddFacilitiesJournalMaskColumn < ActiveRecord::Migration
   def self.up
-    add_column :facilities, :journal_mask, :string, :limit => 50, :null => true, :after => 
+    add_column :facilities, :journal_mask, :string, :limit => 50, :null => true #TODO why? , :after =>
```

Comment:
TIL on top of a TODO: Rails 3.2's `rake notes` does not look for annotations under `db/` but Rails 4 does.